apps/admin-gui/src/app/shared/components/dialogs/add-edit-notification-dialog/tag-section/tag-section.component.html

```diff
@@ -3,13 +3,13 @@
     <button
       (click)="addTag('{' + tag[0] + '}')"
       matTooltip="{{'DIALOGS.NOTIFICATIONS_ADD_EDIT_MAIL.TAG_HINT' | translate }}"
-      class="tag-text w-25"
-      mat-stroked-button
+      class="tag-text w-5"
+      mat-button
```

Comment:
Icon button for nicer look, centered the button in the row and added right margin to give it some space.
```suggestion
      class="tag-text w-5 mt-auto mb-auto me-2"
      mat-icon-button
```